tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "allowJs": true,
```

Comment:
We do turn on `noImplicitAny`, though 😣

Yeah. The tests really shouldn’t be ported. I don’t think we’ll need two `tsconfig.json` files, but if you can get us running with two we can see about shuffling them around 👍